### PR TITLE
feat: add repo organisation files

### PR DESCRIPTION
**Pull request by ndom91**
Add the normal pull request template, labeler.yml, CODE_OF_CONDUCT, etc. files to setup the repo :raised_hands: 

~**EDIT**: Wait a sec on merging this! Still have 1 or 2 small thigns incoming :wink:~ Good to go!

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+merge of your pull request!
+-->
+
+## Reasoning 💡
+
+<!--
+What changes are being made? What feature/bug is being fixed here?
+
+IMPORTANT: Which, if any, adapter is being affected? Or are you wanting to add
+a new one?
```
Comment (balazsorban44) [inline .github/PULL_REQUEST_TEMPLATE.md:23]:
```suggestion
```
Sorry, I missed that in my previous review. This whole template could probably be simplified a lot. Doc changes are not that important to follow a template in my opinion.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+A typical conventional commit looks like this:
+
+```
+type(scope): title
+
+body
+```
+
+Scope is the part that will help grouping the various commit types in the release notes.
+
```
Comment (balazsorban44) [inline CONTRIBUTING.md:41]:
```suggestion
```

I don't think this part is necessary, docs is not subject to semver, so I don't actually care about commit formats on the repo. I guess we can do it anyway to be nice, but not necessary to require it.

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Some additional suggestions.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ndom91) [reply CONTRIBUTING.md]: Yeah good call :+1:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ndom91) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yup, simplified this as well :+1:

**Review by balazsorban44 — APPROVED** [review]
LGTM!

**Pull request merged**